records.py

```diff
@@ -165,6 +165,11 @@ def dataset(self):
         # Create a new Tablib Dataset.
         data = tablib.Dataset()
 
+        # If the RecordCollection is empty, just return the empty set
+        # Check number of rows by typecasting to list
+        if len(list(self)) == 0:
```

Comment:
Fair enough. However, the only way Python evaluates a list to False is when it's empty. With that in mind it's pretty clear/readable what `not list(...)` means and so the check for `len(...) == 0` *needless*.